addons/src/createAutoCorrectedDatePipe.js

```diff
@@ -1,9 +1,9 @@
-export default function createAutoCorrectedDatePipe(dateFormat = 'mm dd yyyy') {
+export default function createAutoCorrectedDatePipe(dateFormat = 'mm dd yyyy', minYear = 0, maxYear = 9999) {
```

Comment:
can you make this an "config" options object (take a look at `createNumberMask` and `emailMail` to see how its implemented there, and do something similar).

Reply:
Also, you've changed the default min value for `yyyy` to `0`, where it was previously `1`.. 

1 seems like a sensible default to me